.github/workflows/style.yml

```diff
@@ -20,11 +20,11 @@ jobs:
         with:
           python-version: ${{ matrix.python }}
           architecture: x64
-      - name: Run flake8
+      - name: Run ruff
```

Comment:
Thanks! Let's rename the job on line 10 as well :pray:

Reply:
Good catch i totally missed this :D